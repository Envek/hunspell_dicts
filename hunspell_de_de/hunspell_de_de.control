# hunspell_de_de extension
comment = 'German Hunspell Dictionary'
default_version = '1.0'
module_pathname = '$libdir/hunspell_de_de'
relocatable = true