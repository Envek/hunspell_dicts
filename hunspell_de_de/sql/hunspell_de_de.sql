CREATE EXTENSION hunspell_de_de;

SELECT ts_lexize('german_hunspell', 'überwachung');
SELECT ts_lexize('german_hunspell', 'etikettierung');
SELECT ts_lexize('german_hunspell', 'vernehmen');
SELECT ts_lexize('german_hunspell', 'verteilen');
SELECT ts_lexize('german_hunspell', 'bereits');
SELECT ts_lexize('german_hunspell', 'üblichen');
SELECT ts_lexize('german_hunspell', 'wahrnehmung');
SELECT ts_lexize('german_hunspell', 'maßen');
SELECT ts_lexize('german_hunspell', 'füße');
