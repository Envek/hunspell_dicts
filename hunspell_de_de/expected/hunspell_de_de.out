CREATE EXTENSION hunspell_de_de;
SELECT ts_lexize('german_hunspell', 'überwachung');
  ts_lexize   
--------------
 {überwachen}
(1 row)

SELECT ts_lexize('german_hunspell', 'etikettierung');
   ts_lexize    
----------------
 {etikettieren}
(1 row)

SELECT ts_lexize('german_hunspell', 'vernehmen');
     ts_lexize      
--------------------
 {vernehmen,nehmen}
(1 row)

SELECT ts_lexize('german_hunspell', 'verteilen');
           ts_lexize           
-------------------------------
 {verteilen,teilen,teile,teil}
(1 row)

SELECT ts_lexize('german_hunspell', 'bereits');
 ts_lexize 
-----------
 {bereit}
(1 row)

SELECT ts_lexize('german_hunspell', 'üblichen');
 ts_lexize 
-----------
 {üblich}
(1 row)

SELECT ts_lexize('german_hunspell', 'wahrnehmung');
  ts_lexize   
--------------
 {wahrnehmen}
(1 row)

SELECT ts_lexize('german_hunspell', 'maßen');
 ts_lexize 
-----------
 {maß}
(1 row)

SELECT ts_lexize('german_hunspell', 'füße');
 ts_lexize 
-----------
 {fuß}
(1 row)

